### PR TITLE
MT: Access details

### PR DESCRIPTION
**Pull request by cloveras**

### CODE REVIEW

File: test-environment.md [hunk 1]
```diff
@@ -100,6 +103,31 @@ Other differences between Production and MT:
 - We allow 10,000 incorrect PIN attempts before locking the Vipps user's account
 - There are no settlement reports and no SFTP service.
 
+## Access to the test environment
+
+### Merchants
+
+All merchants with API access have access to the Vipps test environment.
+A test sale unit is automatically created.
```
Comment (flinkgutt) [inline test-environment.md:111]: Maybe add a bit more to the last sentence there:
Something like this perhaps? 
"A test sale unit is automatically created, and the details can be found on the developer page in https://portal.vipps.no/"

File: test-environment.md [hunk 2]
```diff
@@ -100,6 +103,31 @@ Other differences between Production and MT:
...
+* [How to find the API keys](https://vippsas.github.io/vipps-developer-docs/docs/vipps-developers/developer-resources/portal#how-to-find-the-api-keys)
+* [How to create a test sales unit](https://vippsas.github.io/vipps-developer-docs/docs/vipps-developers/developer-resources/portal#how-to-create-a-test-sales-unit)
+
+### Partners
+
+Partners can get access to the test environment by ordering
+"Vipps Logg inn" (the
+[Vipps Login API](https://vippsas.github.io/vipps-developer-docs/docs/APIs/login-api))
+on
+[portal.vipps.no](https://portal.vipps.no)
```
Comment (rebekaburnett) [inline test-environment.md:123]:
```suggestion
[portal.vipps.no](https://portal.vipps.no).
```

File: test-environment.md [hunk 3]
```diff
@@ -100,6 +103,31 @@ Other differences between Production and MT:
...
+and:
```
Comment (rebekaburnett) [inline test-environment.md:124]:
```suggestion

```

File: test-environment.md (hunk 2 above)
Comment (rebekaburnett) [inline test-environment.md:123]:
```suggestion
[portal.vipps.no](https://portal.vipps.no).
```

File: test-environment.md (hunk 3 above)
Comment (rebekaburnett) [inline test-environment.md:124]:
```suggestion

```

File: test-environment.md (hunk 1 above)
Comment (cloveras) [reply test-environment.md:111]: Good idea!

File: test-environment.md [hunk 4]
```diff
@@ -100,6 +103,31 @@ Other differences between Production and MT:
...
+
+All merchants with API access have access to the Vipps test environment.
+A test sale unit is automatically created.
+
+See:
+* [How to find the API keys](https://vippsas.github.io/vipps-developer-docs/docs/vipps-developers/developer-resources/portal#how-to-find-the-api-keys)
+* [How to create a test sales unit](https://vippsas.github.io/vipps-developer-docs/docs/vipps-developers/developer-resources/portal#how-to-create-a-test-sales-unit)
+
+### Partners
+
```
[the remark this answers is not in the document]
Comment (cloveras) [reply test-environment.md:122]: Good catch!

**Pull request merged**